policyengine_us/parameters/gov/territories/pr/tax/income/credits/earned_income/max_investment_income.yaml

```diff
@@ -0,0 +1,10 @@
+description: The following is the maximum amount allowed in certain forms of income (i.e. child support payments, interest) to receive the earned income credit.
```

Comment:
description format:
Puerto Rico limits the earned income credit to filers with investment income below this threshold.

Reply:
similar for other descriptions